test/quantization/fx/test_equalize_fx.py

```diff
@@ -36,12 +39,14 @@
 
 qconfig_dict = {"": None,
                 "object_type": [(nn.Linear, default_qconfig),
-                                (F.linear, default_qconfig)]}
+                                (F.linear, default_qconfig),
+                                (nni.LinearReLU, default_qconfig)]}
```

Comment:
I don't think this is the correct way to specify fused ops in the qconfig dict. I learnt from @jerryzh168 that today we always fuse Linear + relu sequences, irrespective of what is specified in qconfig_dict. I think we may need to specify the relu type in `object_type` field.
worth confirming/checking it out.